scripts/collector/sqlserver/dmaSQLServerPerfmonDataset.ps1

```diff
@@ -81,8 +81,14 @@ if (!$current_ts) {
 
 if ($namedInstanceName) {
 	$perfmonLogFile = 'opdb__perfMonLog' + '__' + $operation + '_' + $machinename + '_' + $namedInstanceName + '_' + $current_ts + '.log'
+	if (!$perfmonOutFile) {
```

Comment:
Ok to add this but not necessary because this powershell script is never called on its own.  Its always called from the batch file which provides all of the necessary inputs.

Reply:
yes from the main flow, but this allows `manageSQLServerPerfmonDataset.bat` to work.
otherwise it failed.